docs/en/enterprise-edition/content-collections/application-security/get-started/connect-code-and-build-providers/ci-cd-runs/add-checkov.adoc

```diff
@@ -17,6 +17,8 @@ See https://www.checkov.io/2.Basics/Visualizing%20Checkov%20Output.html[Visualiz
 * Application Security: Projects - 'View' permissions
 * Settings: Providers - ('View' and 'Create') OR ('View' and 'Update') permissions
 
+NOTE: Using `--policy-metadata-filter` and `--policy-metadata-filter-exception` will not work with the Developer role and will require one of the other roles or a custom permission group with the above permissions
```

Comment:
+ Note: The Developer role does not support the use of `--policy-metadata-filter` and `--policy-metadata-filter-`exception. This is because these options require permission to call the policies API, which the Developer role does not have. To use these filters, you must either assign a different role to the user or create a custom permission group that includes the necessary permissions.